pyomeca/frame_dependent.py

```diff
@@ -562,7 +562,12 @@ def check_for_nans(self, threshold_channel=10, threshold_consecutive=5):
             Threshold of tolerated consecutive nans on each channel
         """
         # check if there is nans
-        nans = np.isnan(self)
+        if self.get_nan_idx is not None:
```

Comment:
Is this tag set for sure?

Reply:
Yes, it is part of the `__init__` of the FrameDependent class:

```python
self.get_nan_idx = None
```